packages/framework/attributor/src/mixinAttributor.ts

```diff
@@ -147,17 +145,38 @@ export const mixinAttributor = (Base: typeof ContainerRuntime = ContainerRuntime
 			)) as ContainerRuntimeWithAttributor;
 			runtime.runtimeAttributor = runtimeAttributor as RuntimeAttributor;
 
+			const logger = ChildLogger.create(runtime.logger, "Attributor");
+			const mc = loggerToMonitoringContext(context.taggedLogger);
+
+			const shouldTrackAttribution = mc.config.getBoolean(enableOnNewFileKey) ?? false;
+			if (shouldTrackAttribution) {
+				(context.options.attribution ??= {}).track = true;
```

Comment:
i'd move this back. it's possible options objects are copied at some point, in which case doing it after base container runtime load would be problematic.